README.md

```diff
@@ -3,31 +3,25 @@
 </div>
 <br />
 
-# Getting Started With Schematics
+##### Let the cuckoo accommodate the nest for you.
 
-This repository is a basic Schematic implementation that serves as a starting point to create and publish Schematics to NPM.
+CuckooJS is a scaffolding based on <a href="https://nestjs.com/" target="_blank">NestJS</a>. Inspired by the cuckoo bird, we took advantage of the already created nest and included our own easter eggs, such as:
+- <a href="https://commitlint.js.org/#/" target="_blank">Commitlint</a>
+- <a href="https://www.npmjs.com/package/husky" target="_blank">Husky</a>
+- <a href="https://www.npmjs.com/package/@guidesmiths/license-checker" target="_blank">License checker</a> <img alt="logo" src="https://github.com/guidesmiths/license-checker/blob/6e96b2d9a93e3838931e87f8f63af7f144811689/assets/logo.png?raw=true" width="20px" />
 
-### Testing
+This way, you don't have to worry about including these when initiating a new project. 
 
-To test locally, install `@angular-devkit/schematics-cli` globally and use the `schematics` command line tool. That tool acts the same as the `generate` command of the Angular CLI, but also has a debug mode.
 
-Check the documentation with
 
-```bash
-schematics --help
-```
-
-### Unit Testing
+### <img alt="logo" src="./assets/logo.svg" width="20px" />  How to use
 
-`npm run test` will run the unit tests, using Jasmine as a runner and test framework.
 
-### Publishing
-
-To publish, simply do:
 
 ```bash
```

Comment:
Perhaps we can add a description for the command, such as:

```suggestion
```bash
Creating a new NestJS app with the basic tooling:
```